src/src/tx_main.cpp

```diff
@@ -603,13 +602,15 @@ static void CheckConfigChangePending()
     while (pauseCycles--)
       timerCallbackIdle();
 #endif
+    // Prevent any other RF SPI traffic during the commit from RX or scheduled TX
     hwTimer.callbackTock = &timerCallbackIdle;
-    // If telemetry expected in the next interval, the radio is in RX mode
-    // and will skip sending the next packet when the tiemr resumes.
+    // If telemetry expected in the next interval, the radio was in RX mode
+    // and will skip sending the next packet when the timer resumes.
     // Return to normal send mode because if the skipped packet happened
     // to be on the last slot of the FHSS the skip will prevent FHSS
     if (TelemetryRcvPhase == ttrpInReceiveMode)
     {
+      Radio.SetTxIdleMode();
       TelemetryRcvPhase = ttrpTransmitting;
```

Comment:
Any reason not to add `TelemetryRcvPhase == ttrpTransmitting` to the above `if` condition?  Then this `if (TelemetryRcvPhase == ttrpInReceiveMode)` section can be removed and we remove an additional spi call.

`if ((config.IsModified() || ModelUpdatePending) && TelemetryRcvPhase == ttrpTransmitting)`

Reply:
No we can't test it before we disconnect the Tock handler. The ISR can change `TelemetryRcvPhase` after it is tested and our code will start to run expecting that it is in TX mode when by the time we start making changes it is already in RX mode.